### ISSUE TITLE
[Bug]: Program Crashes when Quota is exceeded.

### ISSUE THREAD

**Issue by WesternGamer** [ThioJoe/YouTube-Spammer-Purge#180]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I was scanning all of the channels that I am subscribed to and after about 20 channels, the app crashed. It was due to this error:
`googleapiclient.errors.HttpError: <HttpError 403 when requesting https://youtube.googleapis.com/youtube/v3/search?part=snippet&q=TheCrysisYT&maxResults=1&alt=json returned "The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.". Details: "[{'message': 'The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.', 'domain': 'youtube.quota', 'reason': 'quotaExceeded'}]">`

### Release version

v2.2.3

### Steps to reproduce

1.  Scan channels repeatedly.
2.  Wait until an error appears.
3. Error should be related to `quotaExceeded`.

### What platform are you seeing this problem on?

Windows

### Relevant log output

```shell
`Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 2764, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 2203, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 1051, in validate_channel_id
  File "googleapiclient\_helpers.py", line 131, in positional_wrapper
  File "googleapiclient\http.py", line 937, in execute
googleapiclient.errors.HttpError: <HttpError 403 when requesting https://youtube.googleapis.com/youtube/v3/search?part=snippet&q=TheCrysisYT&maxResults=1&alt=json returned "The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.". Details: "[{'message': 'The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.', 'domain': 'youtube.quota', 'reason': 'quotaExceeded'}]">
[5520] Failed to execute script 'YouTubeSpammerPurge' due to unhandled exception!`
```

**Comment by ThioJoe** [comment]
Ok, I've added some exception handling for when this happens, so it will show a message about what happened and some tips.

It should be in the next release 2.2.5 or whatever is next.

**Comment by ThioJoe** [comment]
Ok now should be fixed with 2.2.5 release

**Comment by WesternGamer** [comment]
Thanks!
